README.adoc

```diff
@@ -189,36 +209,32 @@ Run the following command to start the Appsody development environment:
 appsody run
 ----
 
-The Appsody CLI launches a local Docker container that compiles and hosts the microservice. After some time, you see a message similar to the following example:
+The Appsody CLI launches a local Docker image that contains an Apache Tomcat server that hosts the microservice.
+After some time, you see a message similar to the following example:
+
 [source, role='no_copy']
 ----
 [Container] 2019-09-12 17:28:44.066  INFO 171 --- [  restartedMain] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 4 endpoint(s) beneath base path '/actuator'
 [Container] 2019-09-12 17:28:44.205  INFO 171 --- [  restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
 [Container] 2019-09-12 17:28:44.209  INFO 171 --- [  restartedMain] application.Main                         : Started Main in 6.051 seconds (JVM running for 6.923)
 ----
 
-This message indicates that the Tomcat server is started and you are ready to begin development.
+This message indicates that the Tomcat server is started. Browser to http://localhost:8080 and you can see the Appsody splashscreen.
```

Comment:
Should "splashscreen" be "splash screen"?

Reply:
Yes, agreed. Just spotted a typo too ... browser / browse. Will fix up.

Thank you!